modular_skyrat/code/datums/components/crafting/recipes/recipes_misc.dm

```diff
@@ -20,3 +20,66 @@
 	time = 40
 	subcategory = CAT_TOOL
 	category = CAT_MISC
+
+/datum/crafting_recipe/makeshift/crowbar
+	name = "Makeshift Crowbar"
```

Comment:
```suggestion
	name = "makeshift crowbar"
```
consistency

Reply:
im an idiot, this is recipes. Disregard all of these changes to recipe name changes.